First2.0.Application/Models/HistoricoModel/HistoricoModelBase.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace First2._0.Application.Models.HistoricoModel
+{
+    public class HistoricoModelBase
```

Comment:
Como conversado no Grupo não usaremos mais a Model e sim Dto
```suggestion
    public class HistoricoDtoBase
```